### DISCUSSION TITLE
How do I load an existing model?

### DISCUSSION THREAD

**Discussion by Torprim** [TheLastBen/fast-stable-diffusion#2021] [Q&A]
I uploaded a file onto my google drive, and copied path to the "model download" cell. Launched the cell, and said it detected 1.5 rather than the actual model. I got to the test training steps, and said it didn't detect any models. And from here, I had to manually input the path in which it finally worked. I want to train the model, but not sure how it would work if this problem persists. Am i doing something wrong?

**Comment by TheLastBen** [comment]
the version of your custom model is 1.5, it doesn't mean it's the default original 1.5 model
